### PR TITLE
Return specific error message for duplicated fields in edit_s command

### PR DESCRIPTION
**Pull request by YANYAN-w**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -176,7 +176,8 @@ Edits the details of a student.
 1. `edit_s` edits the student identified by the index. It only accepts a valid index
    based on the list when `list_s` is called.
 2. Tags are replaced if included in the argument, not appended.
-3. For information on the constraints for each parameter used in this command, go
+3. Tags are removed and by entering `t/` without specifying any tags after it.
```
Comment (ghos7ie) [inline docs/UserGuide.md:179]: drop the 'and'

File: src/main/java/seedu/address/logic/commands/editcommands/EditStudentCommand.java [hunk 2]
```diff
@@ -86,24 +92,62 @@ public CommandResult execute(Model model) throws CommandException {
...
+                + MESSAGE_DUPLICATE_FIELD_WARNING + "\n" + String.join("\n", errorMessages),
+                LIST_STUDENT_MARKER);
+        }
         return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedStudent)),
             LIST_STUDENT_MARKER);
     }
 
+    private String getDuplicatedTags(Set<Tag> currentTags, Set<Tag> newTags) {
+        StringBuilder duplicatedTags = new StringBuilder();
+        Set<Tag> intersection = new HashSet<>(currentTags);
```
Comment (ghos7ie) [inline src/main/java/seedu/address/logic/commands/editcommands/EditStudentCommand.java:139]: looks like you are appending the tags, you can remove the line i added

File: docs/UserGuide.md [hunk 3]
```diff
@@ -176,7 +176,8 @@ Edits the details of a student.
 1. `edit_s` edits the student identified by the index. It only accepts a valid index
    based on the list when `list_s` is called.
 2. Tags are replaced if included in the argument, not appended.
```
Comment (ghos7ie) [inline docs/UserGuide.md:178]: This one can remove if you included appending

**Review by ghos7ie — CHANGES_REQUESTED** [review]
Just resolve the comments I have. 

Everything else looks good.

File: docs/UserGuide.md (hunk 3 above)
Comment (ghos7ie) [reply docs/UserGuide.md:178]: whoops, point number 2 can remove if you are intersecting all the tags

Comment (YANYAN-w) [reply docs/UserGuide.md:178]: The intersection is only used to check the presence of duplicated tags, the logic is still to replace the whole tag set. the only change is to add an additional warning about unchanged parts

Comment (ghos7ie) [reply docs/UserGuide.md:178]: ok got it 👍🏻

**Pull request merged**